extensions/Lily/HackedBlocks.js

```diff
@@ -1,7 +1,8 @@
-// Name: Hacked Block Collection
+// Name: Hidden Block Collection
 // ID: lmsHackedBlocks
-// Description: Various modified vanilla blocks.
+// Description: Various vanilla blocks not visible in the block palette.
 // By: LilyMakesThings <https://scratch.mit.edu/users/LilyMakesThings/>
+// Contributors: pumpkinhasapatch

```

Comment:
Do `// By: yourusername` so you get put in the list

Reply:
Is it possible to list more than one name? I can't see any extensions that show multiple contributors over at https://extensions.turbowarp.org/

Reply:
It is! No one's really done it before

Reply:
Well we should probably do the same for [CST1229 in More Events](https://github.com/TurboWarp/extensions/blob/f8182b10b98d5fbb6f57d97c4dbb3c76db12f46a/extensions/Lily/MoreEvents.js#L84) and you in [Scratch Lab Animated Text](https://github.com/TurboWarp/extensions/blob/f8182b10b98d5fbb6f57d97c4dbb3c76db12f46a/extensions/lab/text.js#L679).

Reply:
I don't really about credit for Animated Text, and if I did that it'd look like I made the extension. 

@CST1229?

Reply:
idk, i don't really care